locations/spiders/quiznos.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import re
+import traceback
+
+from locations.items import GeojsonPointItem
+
+
+class QuiznosSpider(scrapy.Spider):
+    name = "quiznos"
+    allowed_domains = ["restaurants.quiznos.com"]
+    start_urls = (
+        'http://restaurants.quiznos.com/data/stores.json?callback=storeList',
+    )
+
+    def parse(self, response):
+
+        try:
+            data = response.body_as_unicode()
+            stores = json.loads(re.search('storeList\((.*)\)', data).group(1))
+        except:
```

Comment:
Please don't catch exceptions like this. I'd prefer to have them bubble up and cause the spider to stop.

Reply:
Don't you need statements for validation?

Reply:
Not in this case. I'd rather the spider throw an exception and have it get logged through the normal scrapy logging system than have it get caught and logged separately.